data/synapse-data-jdbc/src/main/java/io/americanexpress/synapse/data/jdbc/config/SynapseJdbcDataConfig.java

```diff
@@ -0,0 +1,23 @@
+package io.americanexpress.synapse.data.jdbc.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+@Configuration
+@ComponentScan(basePackages = "io.americanexpress.synapse.data.jpa")
```

Comment:
```suggestion
@ComponentScan(basePackages = "io.americanexpress.synapse.data.jdbc")
```
Could you confirm the base package name?